learntools/embeddings/nbs/4-tsne.ipynb

```diff
@@ -4,9 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the previous lesson, we looked at some examples of the movie embeddings we learned, measured the distances between pairs of movies, looked up movies most similar to certain movies, and had some fun composing the semantics of movies with vector math. These are all great ways to debug an embedding model, or understand how it works. But it's also pretty time-consuming.\n",
+    "# Intro\n",
```

Comment:
How strongly do you feel about having these introduction headings at the top of tutorial notebooks? I'm inclined to think it's redundant.

(I checked some of the Python tutorials to see what I did there. Many of them start with an h1 that repeats the kernel title ([example](https://www.kaggle.com/colinmorris/functions-and-getting-help)). Sort of lukewarm on this. It also feels redundant, but the notebook viewer header is a bit visually crowded, so I could see readers missing the title there.)